src/app/components/next-up/next-up.component.scss

```diff
@@ -0,0 +1,83 @@
+.outline {
+    border-style: solid;
+    border-width: 1px;
+    border-color: rgba(0, 0, 0, 0.12);
+  }
+
+.background {
+  background-color: darken($color: #000000, $amount: 80);
```

Comment:
This is interesting, where did you find out about it?  with this and the font color, I think we should go with defaults and let the theme in style.scss decide what they will be.  We can make changes to that later and it will ensure everything is consistent

Reply:
You know, I can't remember where I found that. I was searching how to darken an item in css and angular and I came across it some where.